src/actions.rs

```diff
@@ -83,6 +86,17 @@ impl<'a> Action for Step<'a> {
     async fn call(&self) -> String {
         let gh = GithubClient::new_with_default_token(Client::new());
 
+        // retrieve all Rust compiler meetings
+        // from tomorrow until the next week
+        let tomorrow: chrono::DateTime<chrono::Local> = chrono::Local::now() + Duration::days(1);
+        let h = HttpClientImpl {};
+        let tcompiler_meetings: Result<Vec<CompilerMeeting>, anyhow::Error> = h
+            .get_meetings(
+                tomorrow.format("%F").to_string(),
+                (tomorrow + Duration::days(7)).format("%F").to_string(),
+            )
+            .await;
```

Comment:
I'm not sure about the best way to get the "following weeks' meetings" because it depends on when the agenda generation is run. I'm also hesitant to put fixed dates calculation (such as "get the meetings from this week's Friday until the next Friday").

Suggestions?

Reply:
Always the generation of the agenda depends on when you run it, you get different stuff because issues changes and things like that, so I wouldn't worry that much about this. Unsure if you want tomorrow or today, given that you may also run the script right before the meeting to adjust things.